.github/workflows/deploy-docs.yaml

```diff
@@ -20,11 +21,10 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - name: Install mdbook
-        run: |
-          mkdir mdbook
-          curl -Lf https://github.com/rust-lang/mdBook/releases/download/v0.4.37/mdbook-v0.4.37-x86_64-unknown-linux-gnu.tar.gz | tar -xz --directory=./mdbook
-          echo "`pwd`/mdbook" >> $GITHUB_PATH
+      - name: Install taplo
```

Comment:
```suggestion
      - name: Install mdbook
```

Reply:
@ChrisDenton Yup that's a typo on my side. Will fix.